src/compressor/Compressor.h

```diff
@@ -15,13 +15,18 @@
 #ifndef CEPH_COMPRESSOR_H
 #define CEPH_COMPRESSOR_H
 
-#include <string>
+
 #include <boost/optional.hpp>
-#include "include/memory.h"
+#include <memory>
+#include <string>
+
 #include "include/buffer.h"
+#include "include/int_types.h"
+#include "include/memory.h"
```

Comment:
i think we can ditch `include/memory.h` now, since all the goodies in TR in that header are included in C++11 standard.